### PR TITLE
docs: Added docs for Bikram Sambat library using nextra

### PR DESCRIPTION
**Pull request by thebishalniroula**
### Description
Docs for Bikram Sambat Library using nextra

### CODE REVIEW

File: docs/LICENSE
Comment (ashiishme) [inline docs/LICENSE:1]: Can you remove this file, we don't need it inside docs.

File: docs/README.md
Comment (ashiishme) [inline docs/README.md:1]: Same as above.

File: docs/next.config.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const withNextra = require('nextra')({
+  theme: 'nextra-theme-docs',
+  themeConfig: './theme.config.tsx',
```
Comment (ashiishme) [inline docs/next.config.js:3]: Make sure the theme color reflects Ask Buddie brand color in light mode.

File: docs/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "bikram-sambat-docs",
+  "version": "0.0.1",
+  "description": "Docs for @askbuddie/bikram-sambat",
```
Comment (ashiishme) [inline docs/package.json]:
```suggestion
  "description": "Documentation for Bikram Sambat Nepali date picker by Ask Buddie",
```

File: docs/theme.config.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { DocsThemeConfig } from "nextra-theme-docs";
+
+const config: DocsThemeConfig = {
+  logo: <span>Bikram sambat</span>,
```
Comment (ashiishme) [inline docs/theme.config.tsx]:
```suggestion
  logo: <span>Bikram Sambat</span>,
```

File: docs/theme.config.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  project: {
+    link: "https://github.com/askbuddie/bikram-sambat",
+  },
+  chat: {
+    link: "https://discord.com/channels/735870405709135924",
```
Comment (ashiishme) [inline docs/theme.config.tsx]:
```suggestion
    link: "https://dsc.gg/askbuddie",
```

File: docs/theme.config.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  docsRepositoryBase: "https://github.com/askbuddie/bikram-sambat-docs",
```
Comment (ashiishme) [inline docs/theme.config.tsx]:
```suggestion
  docsRepositoryBase: "https://github.com/askbuddie/bikram-sambat",
```

File: docs/theme.config.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  footer: {
+    text: "Bikram Sambat by Askbuddie",
```
Comment (ashiishme) [inline docs/theme.config.tsx]:
```suggestion
    text: "Bikram Sambat by Ask Buddie",
```

File: src/BikramSambat.test.ts
Comment (ashiishme) [inline src/BikramSambat.test.ts:1]: Revert this file, seems like the changes are coming from the your prettier config.

File: src/BikramSambat.ts
Comment (ashiishme) [inline src/BikramSambat.ts:1]: Same as above, revert this file.

File: docs/next.config.js (hunk 1 above)
Comment (asujan205) [reply docs/next.config.js:3]: change primaryHue value in theme.config,tsx

File: docs/pages/comparing-dates/is-after.mdx [hunk 7]
```diff
@@ -0,0 +1,9 @@
+### `isAfter(date: BikramSambat): boolean`
+
+Returns `true` if the current date is after the provided Bikram Sambat date.
```
Comment (blackkspydo) [inline docs/pages/comparing-dates/is-after.mdx]:
```suggestion
Returns `true` if the provided Bikram Sambat date is after the current date.
```

File: docs/pages/comparing-dates/is-after.mdx [hunk 8]
```diff
@@ -0,0 +1,9 @@
...
+
+```jsx
+const date1 = new BikramSambat("2078-05-15");
+const date2 = new BikramSambat("2078-04-15");
```
Comment (blackkspydo) [inline docs/pages/comparing-dates/is-after.mdx]:
```suggestion
const date2 = new BikramSambat("2078-06-15");
```

File: docs/pages/comparing-dates/is-before.mdx [hunk 9]
```diff
@@ -0,0 +1,9 @@
+### `isBefore(date: BikramSambat): boolean`
+
+Returns `true` if the current date is before the provided Bikram Sambat date.
```
Comment (blackkspydo) [inline docs/pages/comparing-dates/is-before.mdx]:
```suggestion
Returns `true` if the provided Bikram Sambat date is before tha current date.
```

File: docs/pages/comparing-dates/is-before.mdx [hunk 10]
```diff
@@ -0,0 +1,9 @@
...
+
+```jsx
+const date1 = new BikramSambat("2078-05-15");
+const date2 = new BikramSambat("2078-06-15");
```
Comment (blackkspydo) [inline docs/pages/comparing-dates/is-before.mdx]:
```suggestion
const date2 = new BikramSambat("2078-04-15");
```

File: docs/pages/formating-dates/format.mdx [hunk 11]
```diff
@@ -0,0 +1,8 @@
+### `format(dateFormat: DateFormat): string`
+
+Formats the Bikram Sambat date into the specified format.
+
```
Comment (blackkspydo) [inline docs/pages/formating-dates/format.mdx:4]: can you add this block please

The supported date formats are:

```jsx
- YYYY  // 2080
- YYYY-MM // 2080-12
- YYYY-MM-DD  // 2080-12-31
- YYY-MM-DD // 080-12-31
- YY-MM-DD  // 80-12-31
- DD-MMMM-YYYY  // 31-Chaitra-2080
- DD-MM-YYYY  // 31-12-2080
- MMMM-DD-YYYY  // Chaitra-31-2080
- MMMM DD,YYYY  // Chaitra 31,2080
```

Note: The date format can be separated by any of the following characters: `-`, `/`, `(space)`, `(dot)`.

**Comment by thebishalniroula** [comment]
@ashiishme dai, should have been fixed.

File: docs/pages/navigating-dates/get-previous-year.mdx
Comment (blackkspydo) [inline docs/pages/navigating-dates/get-previous-year.mdx:1]: why is this file empty

File: docs/pages/navigating-dates/is-leap-year.mdx [hunk 12]
```diff
@@ -0,0 +1,10 @@
+### `isLeapYear(): boolean`
+
+Returns `true` if the year of the Bikram Sambat date is a leap year.
+
+```jsx
+const date = new Bikram();
+
+Sambat("2078-05-15");
+const isLeap = date.isLeapYear(); // false (Example value)
```
Comment (blackkspydo) [inline docs/pages/navigating-dates/is-leap-year.mdx]:
```suggestion
const date = new BikramSambat("2078-05-15");
const isLeap = date.isLeapYear(); // false (Example value)
```

File: docs/pages/properties/day.mdx
Comment (blackkspydo) [inline docs/pages/properties/day.mdx:1]: let's remove this file, properties are internal use only

File: docs/pages/properties/month.mdx
Comment (blackkspydo) [inline docs/pages/properties/month.mdx:1]: let's remove this file, properties are internal use only

File: docs/pages/properties/year.mdx
Comment (blackkspydo) [inline docs/pages/properties/year.mdx:1]: let's remove this file, properties are internal use only

File: docs/pages/table-of-contents.mdx [hunk 13]
```diff
@@ -0,0 +1,53 @@
+## Table of Contents
+
+- [Introduction](/introduction)
+- [Constructor](/constructor)
+- [Properties](/properties)
+  - [year](/properties/year)
+  - [month](properties/month)
+  - [day](properties/day)
```
Comment (blackkspydo) [inline docs/pages/table-of-contents.mdx]: lets remove this too

**Review by blackkspydo — CHANGES_REQUESTED** [review]

**Review by blackkspydo — APPROVED** [review]

**Pull request merged**